src/math/MathStringUtils.cpp

```diff
@@ -17,7 +18,7 @@ float MathStringUtils::ParseFloat(const std::string& valueString)
 CVector2 MathStringUtils::ParseVector2(const std::string& vectorString)
 {
 	std::vector<std::string> components;
-	boost::split(components, vectorString, boost::is_any_of(", "), boost::algorithm::token_compress_on);
+	StringUtils::split(components, vectorString, ',', true);

```

Comment:
will this be a problem? we're now splitting on `,` but since we're trimming the sides, any space will be removed.

this can be problematic if the line we're splitting contains `,` and `, `(note space after comma)

Reply:
I think it should be alright. I'll make a test to verify.

Reply:
i ran the tests on `stringutils_tests` branch along the latest commit (c745f77), and passed all the tests